src/wp-admin/includes/theme.php

```diff
@@ -1028,18 +1028,18 @@ function customize_themes_print_templates() {
 							/* translators: %s: Theme name. */
 							$aria_label = sprintf( _x( 'Activate %s', 'theme' ), '{{ data.name }}' );
 						?>
-						<# if ( data.compatibleWP && data.compatiblePHP && data.actions.activate ) { #>
+						<# if ( data.compatibleWP && data.compatiblePHP && data.actions.activate && data.compatibleCP !== false ) { #>
 							<a href="{{{ data.actions.activate }}}" class="button button-primary activate" aria-label="<?php echo esc_attr( $aria_label ); ?>"><?php _e( 'Activate' ); ?></a>
 						<# } #>
 					<# } else { #>
-						<# if ( data.compatibleWP && data.compatiblePHP ) { #>
+						<# if ( data.compatibleWP && data.compatiblePHP && data.compatibleCP !== false ) { #>
 							<button type="button" class="button button-primary preview-theme" data-slug="{{ data.id }}"><?php _e( 'Live Preview' ); ?></button>
 						<# } else { #>
 							<button class="button button-primary disabled"><?php _e( 'Live Preview' ); ?></button>
 						<# } #>
 					<# } #>
 				<# } else { #>
-					<# if ( data.compatibleWP && data.compatiblePHP ) { #>
+					<# if ( data.compatibleWP && data.compatiblePHP && data.compatibleCP !== false ) { #>
```

Comment:
For consistency should we try to use the same variable name?

This Pr has a mixture of `data.compatibleCP` and `data.compatible_cp`.
I think I prefer the latter.

Reply:
The mixture is due to the fact that customizer reinvented the wheel when implementing theme installation.
So `data.compatiblePHP` is for customizer and `data.compatible_php` is for standard theme installation.

Reply:
I see now - looks consistent within the file due to upstream inconsistency.